errai-config/src/main/java/org/jboss/errai/config/rebind/EnvUtil.java

```diff
@@ -296,11 +296,19 @@ private static void addSerializableTypes(final Set<MetaClass> exposedClasses, fi
       MetaClassFactory
         .getAllCachedClasses()
         .stream()
-        .filter(mc -> filter.apply(mc.getFullyQualifiedName()))
+        .filter(mc -> filter.apply(mc.getFullyQualifiedName()) && validateWildcardSerializable(mc))
         .collect(toCollection(() -> exposedClasses));
     }
   }
 
+  private static boolean validateWildcardSerializable(MetaClass mc) {
+    if(mc.isInterface() || (mc.isAbstract() && !mc.isEnum())) {
+      log.info("Serializable types cannot be an interface or abstract, ignoring: " + mc.getFullyQualifiedName());
```

Comment:
We probably don't want to spam the logs with types that are non serializable. Could you please change this to debug level?

Also, to the unnecessary String concatenation when this log message isn't printed, you can do this:

    log.debug("Serializable types cannot be an interface or abstract, ignoring: {}", mc.getFullyQualifiedName());